### DISCUSSION TITLE
Push notifications

### DISCUSSION THREAD

**Discussion by manuthecoder** [forem/forem#16876] [Feature Requests]
Push notifications are a great way to keep users engaged to content. Why not implement this in Forem?

**Comment by ellativity** [comment]
I receive push notifications on the Forem app - available for iOS and coming soon for Android. 

@ManuTheCoder are you already using the app?

**Reply by manuthecoder** [reply]
No. This post was intended for the browser version

**Comment by ildi-x** [comment]
One thing I have noticed but ignored until today is the fact that when I updated my iOS to version 15 (currently im on 15.3.1), Forem notification settings have disappeared. Im using an iPhone X. I don't know if this is an issue with the Forem app or if it's a bug on Apple's side of things. I did notice that this happened to a few other apps as well. I have not tried to delete the Forem app and reinstall it again, but that will be my next step and ill report back on here. 

I should also mention that Forem app is no where to be found in `settings > notifications` which I find very strange. 

![forem-app-notifications](https://user-images.githubusercontent.com/24465549/158248586-3615c07a-a843-4e2b-b196-936e4fd45a89.jpeg)

**Reply by ildi-x** [reply]
I can confirm that deleting and re-installing the Forem app fixed this issue for me. Notification settings for Forem have magically reappeared after reinstalling 😇
